Sources/SwiftShell/Shell.swift

```diff
@@ -15,6 +15,20 @@
 //  limitations under the License.
 //
 
-import Foundation
+public protocol Shell {
+    /// The path to the shell command.
+    static var path: String { get }
 
-// TODO: Do stuff here.
+    /// The arguments to the shell command that enable executing an arbitrary command.
+    static var arguments: [String] { get }
+}
+
+public struct ZshShell: Shell {
+    public static let path = "/bin/zsh"
+    public static let arguments = ["-c"]
+}
+
+public struct BashShell: Shell {
+    public static let path = "/bin/bash"
+    public static let arguments = ["-c"]
+}
```

Comment:
This works, another option would be to have a single Shell struct and make 2 properties that are `zsh` and `bash` that are instances of that struct. That's easier to use IMHO at the call site (rather than having to initialize the `ZshShell` and `BashShell` objects). But those are details.

Reply:
well now that you say that out loud... yeah good point 😉 

f43ce312dae9e12e7cede16d8d3c2a93199ed943